main.py

```diff
@@ -1,12 +1,226 @@
+    for choice_num in options:
+        print(f"{options[choice_num]}, type {choice_num}")
+    return options
+
+def make_a_choice(options):
+    valid_choices = options.keys()
+    choice = None
+    while choice not in valid_choices:
+        choice = input(":: ")
+    return choice
+
+def manage_customers():
+    options = {
+        "1": "To list all customers",
+        "2": "To see information for one customer",
+        "3": "To add a new customer",
+        "4": "To edit a customer",
+        "5": "To delete a customer",
+        "6": "To return to the main menu",
+    }
+
+    print("Please choose from the following Customer menu options:")
+    for choice_num in options:
+        print(f"{options[choice_num]}, type {choice_num}")
+    return options
+
+def make_a_choice_in_customer_menu_options(options, store_employee):
+    valid_choices = options.keys()
+    choice = None
+    while choice not in valid_choices:
+        choice = input(":: ")
+    if choice in ['4','5'] and store_employee.selected_customer == None:
+        print("You must select a customer before updating it or deleting it.")
+        print("Let's select a customer!")
+        choice = "2"
+    return choice
+
+def manage_videos():
+    options = {
+        "1": "To list all videos",
+        "2": "To see information for one video",
+        "3": "To add a new video",
+        "4": "To edit a video",
+        "5": "To delete a video",
+        "6": "To return to the main menu",
+    }
+
+    print("Please choose from the following Video menu options:")
+    for choice_num in options:
+        print(f"{options[choice_num]}, type {choice_num}")
+    return options
+
+def make_a_choice_in_video_menu_options(options, store_employee):
+    valid_choices = options.keys()
+    choice = None
+    while choice not in valid_choices:
+        choice = input(":: ")
+    if choice in ['4','5'] and store_employee.selected_video == None:
+        print("You must select a video before updating it or deleting it.")
+        print("Let's select a video!")
+        choice = "2"
+    return choice
+
+def manage_rentals():
+    options = {
+        "1": "To check out a video",
+        "2": "To check in",
+        "3": "To return to the main menu",
+    }
+
+    print("Please choose from the following Rental menu options:")
+    for choice_num in options:
+        print(f"{options[choice_num]}, type {choice_num}")
+    return options
+
+
 def main():
-    print("WELCOME TO RETRO VIDEO STORE")
-    pass
+    store_employee = Employee(url="https://retro-video-store-api.herokuapp.com")
+    
+    quit = False
+    while not quit:
+        print(logo)
+        print_stars()
+        print("WELCOME TO RETRO VIDEO STORE")
+        print_stars()
+        options = list_of_options()
+        print_stars()
+        choice = make_a_choice(options)
+        if choice == "1":
+            exit = False
+            while not exit:
+                print_stars()
```

Comment:
I really appreciate the whole loops within each conditional so that the options are visible to the user after completing a specific option .